crmsh/bootstrap.py

```diff
@@ -531,7 +543,7 @@ def _restart_cluster_on_needed(self):
         else:
             warn("To start sbd.service, need to restart cluster service manually on each node")
```

Comment:
How about 
```
warn("To start sbd.service, need manually restart cluster service on all nodes. Try \" crm cluster run 'crm cluster restart' \" on one node  ")
```

Reply:
I want to change the command as `Try \"crm cluster restart --all\"` after #797 ready

Reply:
> I want to change the command as `Try \"crm cluster restart --all\"` after #797 ready

Yeah. With that, it makes sense to add a TODO code comment inline.